config/settings.yml

```diff
@@ -902,11 +902,16 @@
     - SuspendVM_Task
 :host_scan:
   :queue_timeout: 20.minutes
-:http_proxy:
-  :host:
-  :password:
-  :port:
-  :user:
+ec2_http_proxy:
```

Comment:
Didn't mean to commit these changes to the `settings.yml`, but works as a good example of how I imagine overriding the proxy would work.